wesnoth-multiplayer.po

```diff
@@ -2462,7 +2442,7 @@ msgstr "Scénář, jehož cílem je přežít po co nejdelší dobu."
 #. [side]
 #: data/multiplayer/scenarios/6p_Team_Survival.cfg:261
 msgid "Attacker"
-msgstr "Útočník"
+msgstr "Útočníci"
```

Comment:
proč znovu to množné číslo?

koukal jsem na lobby (čekárnu) a jednotné mi tam sedí - týká se hráče za danou stranu
```suggestion
msgstr "Útočník"
```

Reply:
Viz výše.